help/deploy/complete-setup/configure-auth.md

```diff
@@ -3,75 +3,94 @@ title: "Configure authentication"
 
-1. Go to the Datagrok Settings section 'Users and Sessions'; this section contains all authentication settings.
-2. Enable 'Open Id authentication' to use the OpenID method
-3. Get a well-known-configuration route and set it to 'Open Id Config Endpoint'. It should look
+To setup OpenID authentication:
+
+1. On the left **Sidebar** go to **Settings** > **Users and Sessions**. Authentication settings page appears.
+2. Enable *Open Id authentication* to use the OpenID method
+3. Get a well-known-configuration route and set it to **Open Id Config Endpoint**. It should look
    like `https://login.datagrok.ai/.well-known/openid-configuration`
-4. Set 'Open Id Client Id' and 'Open Id Secret' as in your OpenId provider
-5. Set the' Open Id Code Challenge method' if you enabled authorization code encryption. In most cases, it is `S256`
-6. Set 'Open Id Login Claim', 'Open Id Email Claim', 'Open Id First Name Claim', and 'Open Id Last Name Claim' to
-   provide optional claims for the application
-7. OpenID auto-login can be enabled using the 'Open Id Auto Login' option
-8. Make sure the correct Web Root is set in 'Admin' section
+4. Set *Open Id Client Id* and *Open Id Secret* as in your OpenId provider
+5. Set the *Open Id Code Challenge method* if you enabled authorization code encryption. In most cases, it is `S256`
+6. Set *Open Id Login Claim*, *Open Id Email Claim*, *Open Id First Name Claim*, and *Open Id Last Name Claim* to provide optional claims for the application
+7. You can enable/disable OpenID auto-login using the *Open Id Auto Login* checkbox
+8. Make sure the correct **Web Root** is set in the left **Sidebar** > **Settings** > **Admin** section
 
 ## LDAP authentication
 
-Datagrok can easily integrate with your LDAP server enabling the smooth domain authentication mechanism across all your
+You can integrate Datagrok with your LDAP server by enabling the smooth domain authentication mechanism across all your
 services. You can also use Active Directory to authenticate by this method.
 
-1. Go to the Datagrok Settings section 'Users and Sessions'; this section contains all authentication settings.
-2. Enable 'Domain authentication' to use the LDAP method
-3. Enable 'Domain signup' to enable all users present on a domain controller to authenticate in the Datagrok platform.
-   If the option is disabled, it is required to create the user in the Datagrok platform first to allow the user to log
-   into the platform
+1. On the left **Sidebar** go to **Settings** > **Users and Sessions**. Authentication settings page appears.
+2. Enable *Domain authentication* to use the LDAP method
+3. Enable *Domain signup* to enable all users present on a domain controller to authenticate in the Datagrok platform.
+   If the option is disabled, it is required to create the user in the Datagrok platform first to allow the user to log into the platform
```

Comment:
```suggestion
3. To enable all users on a domain controller to authenticate in the platform, use the *Domain signup* option. Otherwise, you need to create the user in the platform before they can log in.
```